core/Pos/Core/Constants.hs

```diff
@@ -80,23 +80,23 @@ isDevelopment = True
 isDevelopment = False
 #endif
 
--- | System start time embeded into binary.
-staticSysStart :: Maybe Timestamp
+-- | System start time embedded into binary.
+staticSysStart :: Timestamp
 staticSysStart
-    | isDevelopment = Nothing
-    | st > 0        = Just $ Timestamp $ sec st
+    | isDevelopment    = error "System start time should be passed \
+                              \as a command line argument in dev mode."
+    | st >= 1400000000 = Timestamp $ sec st
     -- If several local nodes are started within 20 sec,
     -- they'll have same start time
-    | otherwise     = Just $ Timestamp $ sec $
-          (after3Mins `div` divider + alignment) * divider
+    | otherwise        =
+          let frameLength  = st
```

Comment:
I'm confused about `st`. Here it's used as the frame length, but above it's used as the system start time.